app/dashboard-user/dashboard-user.component.html

```diff
@@ -0,0 +1,34 @@
+<rom-base-view>
+
+  <GridLayout rows="auto, *, auto" columns="*, auto">
+
+    <StackLayout row="0" colSpan="2" class="stats" orientation="horizontal">
+      <rom-tile text="Total records" value="10260"></rom-tile>
+      <rom-tile text="Active users" value="89"></rom-tile>
+      <rom-tile text="Total users" value="500"></rom-tile>
+    </StackLayout>
+
+    <StackLayout row="1" rowSpan="2" col="0">
+
+      <RadCartesianChart class="stroke-chart">
+        <CategoricalAxis tkCartesianHorizontalAxis></CategoricalAxis>
+        <LinearAxis tkCartesianVerticalAxis></LinearAxis>
+        <BarSeries tkCartesianSeries showLabels="true" seriesName="Bar" stackMode="Stack" [items]="source" categoryProperty="Country"
+          valueProperty="Amount">
+        </BarSeries>
+      </RadCartesianChart>
+
+    </StackLayout>
+
+    <StackLayout row="2" col="1" class="buttons side-layout" orientation="vertical">
+      <StackLayout class="side-tile">
```

Comment:
Yeah. Use the `ContentLayout` class like here: https://github.com/racketometer/frontend-application/pull/46/files#diff-5921e875726fa21c5f8911e700c4067bR7